js/index.esm.js

```diff
@@ -16,6 +16,7 @@ import ScrollSpy from './src/scrollspy'
 import Tab from './src/tab'
 import Toast from './src/toast'
 import Tooltip from './src/tooltip'
+import '../node_modules/focus-visible/dist/focus-visible.js'
```

Comment:
That's not how it should be imported BTW :P

Reply:
That's pretty much what I meant:
> I added the dependency, but this definetly require some JS love to be included in the best possible way. For now, I roughly imported the file form the `node_modules`—which can't obviously stay this way.

:D